libpod/boltdb_state.go

```diff
@@ -60,7 +61,11 @@ func NewBoltState(path, lockDir string, runtime *Runtime) (State, error) {
 		if _, err := tx.CreateBucketIfNotExists(allPodsBkt); err != nil {
 			return errors.Wrapf(err, "error creating all pods bucket")
 		}
-		if _, err := tx.CreateBucketIfNotExists(runtimeConfigBkt); err != nil {
+		if bucket, err := tx.CreateBucket(runtimeConfigBkt); err == nil {
+			if err := bucket.Put(osKey, []byte(runtime.GOOS)); err != nil {
```

Comment:
No need for the manual `Put` here - `validateAgainstConfig` will insert any key that is missing, and compare against any key that is present, so it will handle adding osKey if it's not present

Reply:
> No need for the manual `Put` here - `validateAgainstConfig` will...

Oh, I see.  Fixed with 7245d52d -> 912a5e83.